content/docs/command-reference/checkout.md

```diff
@@ -135,7 +135,6 @@ The workspace looks something like this:
 ├── evaluate.dvc
 ├── featurize.dvc
 ├── prepare.dvc
-├── train.dvc
 ├── src
 │   └── ...
 └── train.dvc
```

Comment:
It was probably easier to read before?

Reply:
Do you mean `train.dvc` should be above `src`?  This will have better readability sure.
I just left this same as output (displayed alphabetically).

Reply:
Yes above src. Oh well, Ivan already approved and merged this one so let's go with it. Thanks